dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/gdv/GdvDataController.kt

```diff
@@ -0,0 +1,54 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.gdv
```

Comment:
Do we need some backend tests (unit / e2e) in the backend for the new framework?

Reply:
We are, implicitly, already testing this component during prepopulation. Therefore, I see little value in adding a new dataland-e2etest test (especially since that would probably require refactoring the ApiAccessor.kt component).

Reply:
Fair point. Then lets see what the test coverage looks like in sonar, once CI is green